docs/docs/installation.md

```diff
@@ -101,7 +101,7 @@ If you use [Create React App](https://github.com/facebookincubator/create-react-
 
 #### Webpack
 
-Include both `DefinePlugin` and `UglifyJsPlugin` into your production Webpack configuration as described in [this guide](https://webpack.js.org/guides/production-build/).
+Include both [`DefinePlugin`](https://webpack.github.io/docs/list-of-plugins.html#defineplugin) and [`UglifyJsPlugin`](https://webpack.github.io/docs/list-of-plugins.html#uglifyjsplugin) into your production Webpack configuration.
```

Comment:
In this particular case I'd rather link to 2.x doc because it contains specific actionable advice, and works identically in 1.x.